catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityMarkerPlugin.java

```diff
@@ -104,8 +108,17 @@ public UpdateRequest process(UpdateRequest input)
                 validator.validate(metacard);
             } catch (ValidationException e) {
                 String validatorName = getValidatorName(validator);
+                boolean validationErrorsExist = CollectionUtils.isNotEmpty(e.getErrors());
+                boolean validationWarningsExist = CollectionUtils.isNotEmpty(e.getWarnings());
 
-                if (isValidatorEnforced(validatorName)) {
+                if (isValidatorEnforced(validatorName) && validationErrorsExist && enforceErrors) {
+                    INGEST_LOGGER.info(
```

Comment:
I think this should be debug, because it is for each validator (and each metacard)
I think we should add a aggregrate `info` statement in process, along side the validation warnings and errors output.